### PR TITLE
repart: Add some more logging

### PR DESCRIPTION
**Pull request by daandemeyer**

### CODE REVIEW

File: src/partition/repart.c [hunk 1]
```diff
@@ -5471,9 +5481,9 @@ static int context_minimize(Context *context) {
                 if (r < 0)
                         return log_error_errno(r, "Failed to generate temporary file path: %m");
 
-                if (fstype_is_ro(p->format))
+                if (fstype_is_ro(p->format)) {
```
Comment (bluca) [inline src/partition/repart.c]: no need for these braces

**Comment by bluca** [comment]
Should we have a `--quiet` mode? No need to do it in this PR

File: src/libsystemd/sd-netlink/netlink-socket.c [hunk 2]
```diff
@@ -23,14 +23,22 @@ static int broadcast_groups_get(sd_netlink *nl) {
         assert(nl);
         assert(nl->fd >= 0);
 
+        log_debug("NETLINK_LIST_MEMBERSHIPS");
```
Comment (bluca) [inline src/libsystemd/sd-netlink/netlink-socket.c]: rebase gone wrong?

**Review by bluca — APPROVED** [review]

File: src/shared/mkfs-util.c [hunk 3]
```diff
@@ -479,6 +479,13 @@ int make_filesystem(
         if (extra_mkfs_args && strv_extend_strv(&argv, extra_mkfs_args, false) < 0)
                 return log_oom();
 
+        if (DEBUG_LOGGING) {
+                _cleanup_free_ char *j = NULL;
+
+                j = strv_join(argv, " ");
+                log_debug("Executing mkfs command: %s", j);
```
Comment (poettering) [inline src/shared/mkfs-util.c]: strna()

passing NULL to %s is not OK

**Pull request merged**